projects/aws/eks-a-admin-image/provisioners/al2/install_eksa.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+echo $EKSA_VERSION
+
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# YQ_URL needs to be provided to the script
+
+EKSCTL_VERSION="${EKSCTL_VERSION:-latest}" # The eksctl version to install (example v0.0.0)
+EKSA_RELEASE_MANIFEST_URL="${EKSA_RELEASE_MANIFEST_URL:-https://anywhere-assets.eks.amazonaws.com/releases/eks-a/manifest.yaml}" # The url pointing to the eks-a releases manifest
+EKSA_VERSION="${EKSA_VERSION:-latest}" # The eks-a version to install (example v0.0.0)
+
+# Install depencies
+sudo yum install -y tar
```

Comment:
maybe we could move this to a separate script so we can share the rest of the code between OS's?